asset/doc.css

```diff
@@ -85,3 +85,24 @@ div.odoc div.spec tbody td.def {
   overflow-wrap: anywhere;
   min-width: 40%
 }
+
+.odoc-include details {
+  position: relative;
+}
+
+.odoc-include details:after {
+  display: block;
+  content: " ";
```

Comment:
Why do we need a `content`?

Reply:
Yeah I wasn't sure about this - the previous Tyxml page code has this snippet and it's for the pseudo-element (i.e. `:after`) after all so I thought it would need this part. I'll erase it.

Reply:
Maybe we do need it, it was just a genuine question 🙂  Let's try removing it and see how the docs looks!